src/transform/FooterContainer.js

```diff
@@ -0,0 +1,91 @@
+import './FooterContainer.css'
+import Polyfill from './Polyfill'
+
+/**
+ * Ensures the 'Read more' section header can always be scrolled to the top of the screen.
+ * @param {!Window} window
+ * @return {void}
+ */
+const updateBottomPaddingToAllowReadMoreToScrollToTop = window => {
+  const div = window.document.getElementById('pagelib_footer_container_ensure_can_scroll_to_top')
+  const currentPadding = parseInt(div.style.paddingBottom, 10) || 0
+  const height = div.clientHeight - currentPadding
+  const newPadding = Math.max(0, window.innerHeight - height)
+  div.style.paddingBottom = `${newPadding}px`
+}
+
+/**
+ * Allows native code to adjust footer container margins without having to worry about
+ * implementation details.
+ * @param {!number} margin
+ * @param {!Document} document
+ * @return {void}
+ */
+const updateLeftAndRightMargin = (margin, document) => {
+  const elements = Polyfill.querySelectorAll(document, `
+    #pagelib_footer_container_menu_heading, 
+    #pagelib_footer_container_readmore, 
+    #pagelib_footer_container_legal
+  `)
+  Array.from(elements)
+    .forEach(element => {
+      element.style.marginLeft = `${margin}px`
+      element.style.marginRight = `${margin}px`
+    })
+  const rightOrLeft = document.querySelector('html').dir === 'rtl' ? 'right' : 'left'
+  Array.from(Polyfill.querySelectorAll(document, '.pagelib_footer_menu_item'))
```

Comment:
Since you're using the polyfill which always returns an array, I don't think you need the Array.from any more.